bitgouel-api/src/main/kotlin/team/msg/domain/admin/service/AdminServiceImpl.kt

```diff
@@ -214,6 +231,113 @@ class AdminServiceImpl(
         }
     }
 
+    /**
+     * 강의 리스트 엑셀을 업로드 하는 비지니스 로직입니다
+     * @param file  강의 리스트 엑셀 업로드를 위한 MultipartFile
+     */
+    @Transactional(rollbackFor = [Exception::class])
+    override fun uploadLectureListExcel(file: MultipartFile) {
+        file.inputStream.use {
+            val workbook = try {
+                WorkbookFactory.create(file.inputStream)
```

Comment:
```suggestion
val workbook = file.inputStream.use {
    try {
        WorkbookFactory.create(it)
```

이런 식으로 작성하면 스트림을 한 번만 열어도 될 것 같아요.

Reply:
[d0a7cb2](https://github.com/School-of-Company/Bitgouel-Server/pull/575/commits/d0a7cb23229c9c2b4d38fa53af5438cd9c031ff6) 다른 엑셀 API...도 변경했어요!

Reply:
```kt
        val workbook = file.inputStream.use {
            try {
                WorkbookFactory.create(it)
            } catch (e: IndexOutOfBoundsException) {
                throw InvalidCellTypeException("셀 서식을 텍스트로 변경해주세요.")
            } catch (e: Exception) {
                throw InternalServerException("엑셀 파일 처리 중 문제가 발생했습니다. info : [ errorMessage = ${e.message} ]")
            }
        }
```

제가 의도한 코드는 이런 느낌이었습니다!

Reply:
[4eaba1d](https://github.com/School-of-Company/Bitgouel-Server/pull/575/commits/4eaba1d79f8c194b338de8f7c0383f44d93d6000) 반영헸습니다!